website/members/templates/members/user/edit_profile.html

```diff
@@ -10,12 +10,6 @@
         <div class="container">
             <h1 class="section-title">{% trans "edit profile"|capfirst %}</h1>
 
-            {% if messages %}
-                {% for message in messages %}
-                    {% alert message.tags message %}
-                {% endfor %}
-            {% endif %}
```

Comment:
We no longer need this?

Reply:
I replaced it by {% bootstrap messages %} in the base template

Reply:
Before:

![image](https://user-images.githubusercontent.com/1799914/149666240-efffbabd-3e79-4c05-87cc-d826146f4b7a.png)

Now:

![image](https://user-images.githubusercontent.com/1799914/149666242-69eff283-ad56-4eea-bbf1-f25f6b99f239.png)


Idea is nice, in practice it doesn't improve the UI ;)
Wouldn't do it in this PR tbh.